ddp-workspace/projects/ddp-singular/src/styles/_common.scss

```diff
@@ -9,7 +9,6 @@
 
 
 html {
-  height: 100%;
```

Comment:
Removed for the same reason as https://github.com/broadinstitute/ddp-angular/pull/1628/files#r937932295